utils/mariadb/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if which mysqld 2> /dev/null >&2; then
```

Comment:
surprised shellcheck doesn't complain here. command -v should be used instead of which.